Packs/FraudWatch/Integrations/FraudWatch/FraudWatch.py

```diff
@@ -0,0 +1,854 @@
+    limit = get_and_validate_positive_int_argument(args, 'limit')
+
+    try:
+        raw_response = client.incident_contact_emails_list(incident_id, page, limit)
+    except DemistoException as e:
+        if 'Contact email not found' in str(e):
+            page_error_msg = f'''Make sure page index: {page} is within bounds.''' if page else ''
+            unknown_incident_msg = f'''Make sure incident id: {incident_id} is correct.'''
+            raise DemistoException(
+                f'''Error occurred in command 'fraudwatch-incident-contact-emails-list'. {page_error_msg}'''
+                f' {unknown_incident_msg}')
+        raise e
+
+    outputs = [dict(output, identifier=incident_id) for output in raw_response]
+
+    return CommandResults(
+        outputs_prefix='FraudWatch.IncidentContacts',
+        outputs=outputs,
+        outputs_key_field='noteId',
+        raw_response=raw_response,
+        readable_output=tableToMarkdown("FraudWatch Incident Contacts Data", outputs,
+                                        ['noteId', 'incident_id', 'subject', 'creator', 'content', 'date'],
+                                        removeNull=True)
+    )
+
+
+def fraud_watch_incident_messages_add_command(client: Client, args: Dict):
+    """
+    Add a new message to be associated to the incident which corresponds to the given incident ID:
+    - Incident ID (Required): The ID of the incident to add a message to its email contacts.
+    - Message Content (Required): Content of the message.
+
+    Known errors that causes error to be returned by FraudWatch service:
+    - Unknown incident id.
+
+    Args:
+        client (Client): FraudWatch client to perform the API calls.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    incident_id = args.get('incident_id')
+    message_content = args.get('message_content')
+
+    raw_response = client.incident_messages_add(incident_id, message_content)
+
+    human_readable = f'### Message for incident id {incident_id} was added successfully.'
+
+    return CommandResults(
+        raw_response=raw_response,
+        readable_output=human_readable
+    )
+
+
+def fraud_watch_incident_urls_add_command(client: Client, args: Dict) -> CommandResults:
+    """
+    Adds additional urls to the incident which corresponds to the given incident ID:
+    - Incident ID (Required): The ID of the incident to add additional urls to.
+    - Urls: Additional urls to be added to the incident that matches 'Incident ID'.
+
+    Known errors that causes error to be returned by FraudWatch service:
+    - Unknown incident id.
+
+    Args:
+        client (Client): FraudWatch client to perform the API calls.
+        args (Dict): Demisto arguments.
+
+    Returns:
+        CommandResults.
+    """
+    incident_id = args.get('incident_id')
+    raw_urls = argToList(args.get('urls'))
+
+    urls: Dict[str, List[str]] = {
+        'urls[]': [raw_url for raw_url in raw_urls]
```

Comment:
Why do we need the reference copy? why not just:
```
'urls[]': raw_urls
```

Reply:
Correct. Fixed.